doc/calling-javascript.md

```diff
@@ -443,12 +456,13 @@ class A { def foo(x: Int) = s"Input: $x" }
 print(new A())
 {% endhighlight %}
 Note that `A` does *not* extend `T` but only conforms structurally
-(i.e. it has a method `foo` with corresponding signature).
+(i.e., it has a method `foo` with the same signature).
```

Comment:
no, structural types are covariant in the return type of the methods. So the method may not have the same signature. Feel free to chose another word than corresponding though.